inc/cleantalk-public.php

```diff
@@ -931,25 +931,24 @@ function ct_die_extended($comment_body)
 /**
  * Validates JavaScript anti-spam test
  *
- * @param string $field_name filed to serach in data
- * @param array|null $data Data to search in
+ * @param string $check_js_value String to checking
  * @param bool $is_cookie
  *
  * @return int|null
  */
-function apbct_js_test($field_name = 'ct_checkjs', $data = null, $is_cookie = false)
+function apbct_js_test($check_js_value = '', $is_cookie = false)
```

Comment:
I don't see any reason to leave **is_cookie** flag.

Reply:
Discussed on the chat. Applied.